<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParameterModificationWindow</class>
 <widget class="QWidget" name="ParameterModificationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Parameter modification</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="3">
    <widget class="QPushButton" name="synthesizeToFileButton">
     <property name="text">
      <string>Synthesize to file</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Modification:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="synthesizeButton">
     <property name="text">
      <string>Synthesize</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDoubleSpinBox" name="amplitudeSpinBox">
     <property name="minimum">
      <double>-99.989999999999995</double>
     </property>
     <property name="maximum">
      <double>99.989999999999995</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="addRadioButton">
        <property name="text">
         <string>Add</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="multiplyRadioButton">
        <property name="text">
         <string>&amp;Multiply</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Parameter:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="resetParameterButton">
     <property name="text">
      <string>Reset parameter</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QComboBox" name="outputGainComboBox"/>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="parameterComboBox">
     <property name="currentText">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Amplitude:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QCheckBox" name="saveVTMParamCheckBox">
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>Save VTM parameters to file</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Output gain (dB):</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="4">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Parameter curve</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="Lab::Figure2DWidget" name="parameterCurveWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>200</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Input</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="GS::ParameterModificationWidget" name="parameterModificationWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>300</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>CrossCursor</cursorShape>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Lab::Figure2DWidget</class>
   <extends>QWidget</extends>
   <header>Figure2DWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GS::ParameterModificationWidget</class>
   <extends>QWidget</extends>
   <header>ParameterModificationWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>parameterComboBox</tabstop>
  <tabstop>addRadioButton</tabstop>
  <tabstop>multiplyRadioButton</tabstop>
  <tabstop>amplitudeSpinBox</tabstop>
  <tabstop>outputGainComboBox</tabstop>
  <tabstop>resetParameterButton</tabstop>
  <tabstop>synthesizeButton</tabstop>
  <tabstop>saveVTMParamCheckBox</tabstop>
  <tabstop>synthesizeToFileButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
